src/reactive_state_access.rs

```diff
@@ -670,4 +708,69 @@ mod test {
 
         assert_eq!(a().state_exists(), false, "The a state should not exist");
     }
+    #[test]
+    fn test_reaction() {
+        let a_b_subtraction = a_b_subtraction();
+        a().set(0);
+        b().set(0);
+        a().update(|state| *state = 40);
+        assert_eq!(a().get(), 40, "We should get 40 as value for a");
+        assert_eq!(
+            a_b_subtraction.get(),
+            40,
+            "We should get 40 for subtraction because setting"
+        );
+
+        b().set(10);
+        assert_eq!(
+            a_b_subtraction.get(),
+            30,
+            "We should get 40 for subtraction because setting"
+        );
+        b().inert_set(0);
+        assert_eq!(
+            a_b_subtraction.get(),
+            30,
+            "We should get 30 for subtraction because setting inert"
+        );
+        b().set(20);
+        assert_eq!(
+            a_b_subtraction.get(),
+            20,
+            "We should get 20 for subtraction because setting"
+        );
+    }
+    #[test]
+    fn test_reversible_reaction() {
+        let a_b_reversible_subtraction = a_b_reversible_subtraction();
+        a_reversible().set(0);
+        b_reversible().set(0);
+        a_reversible().update(|state| *state = 40);
+        assert_eq!(a_reversible().get(), 40, "We should get 40 as value for a");
+        assert_eq!(
+            a_b_reversible_subtraction.get(),
+            40,
+            "We should get 40 for subtraction because setting"
+        );
+
+        global_undo_queue().travel_backwards();
```

Comment:
Here the unit test is failing. When traveling back in time, we should get a_reversible() set to 0, so the subtraction should be 0, right ?